examples/counter-hot/store/actions.js

```diff
@@ -1,5 +1,10 @@
-export const increment = ({ commit }) => commit('increment')
-export const decrement = ({ commit }) => commit('decrement')
+export const increment = ({ commit }) => {
+  commit('increment')
+  commit('pushToHistory', 'increment')
```

Comment:
Did you forget to remove this line? 😄

Reply:
Whoops! The tests pass locally so I didn't notice 😆